_pytest/logging.py

```diff
@@ -176,6 +176,10 @@ def emit(self, record):
         self.records.append(record)
         logging.StreamHandler.emit(self, record)
 
+    def reset(self):
```

Comment:
For consistency perhaps we should call this `clear` too?

Reply:
But i see that there pre-existed also `_LiveLoggingStreamHandler.reset()` method,
so maybe @thisch had a reason to use this name.

Reply:
Oh I didn't notice that, never mind then.

Reply:
Note that the docstring of LogCaptureFixure.clear starts with "reset" ;) There is no specific reson why I chose "reset" in my initial commit, though.